examples/dns/README.md

```diff
@@ -0,0 +1,33 @@
+# DNS Smart contract
+
+The DNS smart contract is our showcase for executing other smart contracts on-chain.
+
+   
+
+> This is a port from the blog post's ink! 1.0 based version of the contract to ink! 2.0.
+
+Domain name service contract inspired by
+    [this blog post on medium](https://medium.com/@chainx_org/secure-and-decentralized-polkadot-domain-name-system-e06c35c2a48d).
+
+### Description
+The main function of this contract is domain name resolution which refers to the retrieval of numeric values corresponding to readable and easily memorable names such as "polka.dot" which can be used to facilitate transfers, voting and DApp-related operations instead of resorting to long IP addresses that are hard to remember.
+<br>
+
+![Image](dns_diagram.png)
```

Comment:
Thanks for the diagram. Can you make a `examples/dns/.images` folder and put it in there instead?